framework/src/parser/Syntax.C

```diff
@@ -314,6 +319,15 @@ Syntax::verifyMooseObjectTask(const std::string & base, const std::string & task
     if (task == task_it.second)
       return true;
 
+  iters = _deprecated_moose_systems_to_tasks.equal_range(base);
+  for (const auto & task_it : as_range(iters))
+    if (task == task_it.second)
+    {
+      mooseDeprecated("Adding '" + base + "' though task '" + task +
+                      "' is deprecated. The syntax you are using is deprecated.");
```

Comment:
Second sentence seems redundant

Reply:
changed

Reply:
ok I made it as obvious as possible what is changing / happening in the deprecation message. Maybe even too obvious